docs/reference-guide/triggers.md

```diff
@@ -127,6 +124,7 @@ Based on the event type, the following predefined variables are available:
 | ON --> DELETE | deletedEdges |
```

Comment:
Should we maybe add some examples?

Reply:
Yes, but I do not know some logical example for that at the moment, so I will add it to tasks and update with example later.